collector/project.go

```diff
@@ -49,9 +53,18 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 	var poUtils []float64
 	var hpaUtils []float64
 
+	// Setup vars for catalogs
+	perClusterCatalogMap := make(map[string]bool)
+	rancherCatalog, err := c.Client.Catalog.ByID("library")
```

Comment:
What if the catalog is not named library or there is another catalog named library? You can disable our catalog and name a new catalog library.

Reply:
If another catalog is named library it will fail the url-equals test on line below and we won't track anything. If rancher library isn't named 'library' (or isn't global) this just won't catch it, but thats ok since we are just trying to get a rough idea anyway.

Reply:
Sure, but listing the catalogs and checking their url instead would remove the need for the line underneath and catch the catalog if its named something else. seems like similar effort to me. With that being said, it probably isn't all too common so up to you.

Reply:
Thats true, and we wen't down that path originally (not sure if that was your code or mine). Ended up reverting because it was complicating things. Another point is that if it's still set to defaults it has likely never changed, so library names have not been swapped which limits externalID's being incorrect.